src/latitudes.cpp

```diff
@@ -204,3 +152,211 @@ double pj_authalic_lat_inverse_exact(double beta, const double *APA,
+        61.0/240.0, -103.0/140.0, 15061.0/26880.0, 167603.0/181440.0,
+        49561.0/161280.0, -179.0/168.0, 6601661.0/7257600.0,
+        34729.0/80640.0, -3418889.0/1995840.0,
+        212378941.0/319334400.0,
+        // C[mu,xi] skipped
+        // C[chi,phi]
+        -2.0, 2.0/3.0, 4.0/3.0, -82.0/45.0, 32.0/45.0, 4642.0/4725.0,
+        5.0/3.0, -16.0/15.0, -13.0/9.0, 904.0/315.0, -1522.0/945.0,
+        -26.0/15.0, 34.0/21.0, 8.0/5.0, -12686.0/2835.0,
+        1237.0/630.0, -12.0/5.0, -24832.0/14175.0,
+        -734.0/315.0, 109598.0/31185.0,
+        444337.0/155925.0,
+        // C[chi,beta] skipped
+        // C[chi,theta] skipped
+        // C[chi,mu]
+        -1.0/2.0, 2.0/3.0, -37.0/96.0, 1.0/360.0, 81.0/512.0,
+        -96199.0/604800.0,
+        -1.0/48.0, -1.0/15.0, 437.0/1440.0, -46.0/105.0, 1118711.0/3870720.0,
+        -17.0/480.0, 37.0/840.0, 209.0/4480.0, -5569.0/90720.0,
+        -4397.0/161280.0, 11.0/504.0, 830251.0/7257600.0,
+        -4583.0/161280.0, 108847.0/3991680.0,
+        -20648693.0/638668800.0,
+        // C[chi,chi] skipped
+        // C[chi,xi] skipped
+        // C[xi,phi]
+        -4.0/3.0, -4.0/45.0, 88.0/315.0, 538.0/4725.0, 20824.0/467775.0,
+        -44732.0/2837835.0,
+        34.0/45.0, 8.0/105.0, -2482.0/14175.0, -37192.0/467775.0,
+        -12467764.0/212837625.0,
+        -1532.0/2835.0, -898.0/14175.0, 54968.0/467775.0,
+        100320856.0/1915538625.0,
+        6007.0/14175.0, 24496.0/467775.0, -5884124.0/70945875.0,
+        -23356.0/66825.0, -839792.0/19348875.0,
+        570284222.0/1915538625.0,
+        // C[xi,beta] skipped
+        // C[xi,theta] skipped
+        // C[xi,mu] skipped
+        // C[xi,chi] skipped
+        // C[xi,xi] skipped
+    };
+    static const int ptrs[] = {
+        0, 0, 0, 0, 12, 33, 54, 54, 54, 54, 54, 54, 54, 54, 54, 54, 54, 54, 54,
+        66, 66, 66, 66, 87, 87, 108, 108, 108, 129, 129, 129, 150, 150, 150,
+        150, 150, 150,
+    };
+    if (!( auxin >= AuxLat::START && auxin < AuxLat::AUXNUMBER &&
+           auxout >= AuxLat::START && auxout < AuxLat::AUXNUMBER ))
+        throw std::out_of_range("Bad specification for auxiliary latitude");
+    int k = int(AuxLat::AUXNUMBER) * int(auxout) + int(auxin), o = ptrs[k];
+    if (o == ptrs[k+1])
+        throw std::out_of_range
+            ("Unsupported conversion between auxiliary latitudes");
+    double d = n, n2 = n*n;
+    if (auxin <= AuxLat::RECTIFYING && auxout <= AuxLat::RECTIFYING) {
+        for (int l = 0; l < Lmax; ++l) {
+            int m = (Lmax - l - 1) / 2; // order of polynomial in n^2
+            c[l] = d * pj_polyval(n2, coeffs + o, m);
+            o += m + 1;
+            d *= n;
+        }
+    } else {
+        for (int l = 0; l < Lmax; ++l) {
+            int m = (Lmax - l - 1); // order of polynomial in n
+            c[l] = d * pj_polyval(n, coeffs + o, m);
+            o += m + 1;
+            d *= n;
+        }
+    }
+    // assert (o == ptrs[k+1]);
```

Comment:
Consider removing this

Reply:
You mean the assert?  This provided useful information for the next developer to understand how the ptrs array is used.  Let's keep.